1-js/04-object-basics/07-optional-chaining/article.md

```diff
@@ -166,9 +166,9 @@ userGuest.admin?.(); // nothing (no such method)
 */!*
 ```
 
-Here, in both lines we first use the dot (`user1.admin`) to get `admin` property, because the user object must exist, so it's safe read from it.
+Here, in both lines we first use the dot notion `userAdmin.admin\userGuest.admin` to get `admin` property, because we assume the `userAdmin\userGuest` object must exist, so it's safe read from it.
```

Comment:
A suggestion:

```suggestion
Here, in both lines we first use the dot notation (`userAdmin.admin`) to get `admin` property, because we assume  `userAdmin` and `userGuest` objects must exist, so it's safe to read from it.
```

Reply:
We are mentioning `userAdmin.admin` after "dot notation" to remind users what dot notation is. So there is no need to mention `userGuest`.